cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml

```diff
@@ -150,7 +150,7 @@ spec:
         storage_type: ceph
       expr: |
         label_replace((ceph_osd_in == 1 and ceph_osd_up == 0),"disk","$1","ceph_daemon","osd.(.*)") + on(ceph_daemon) group_left(host, device) label_replace(ceph_disk_occupation,"host","$1","exported_instance","(.*)")
-      for: 1m
+      for: 15m
```

Comment:
changing interval for a critical severity error like `CephOSDDiskNotResponding` from 1 minute to 15 minutes seems a bit ... risky!

Reply:
@aruniiird what's the rationale behind that change?

Reply:
This change is mainly for `ODF Managed Service` product , their SREs (their customer support team) are getting (action required) alerts, which are automatically resolved (either by ceph itself or through OCS Operator reconciliation). So they want to increase the alert time delay for the following alerts,
```
CephMonHighNumberOfLeaderChanges
CephOSDDiskNotResponding
CephClusterWarningState
```

As we don't have a separate alert mechanism for Managed Services , making the changes here.